### ISSUE TITLE
When the window is not focused, telegram is assumed not to be online. Can we have an option not to?

### ISSUE THREAD

**Issue by pouyakary** [overtake/TelegramSwift#619]
Sometimes I want to be able to read something and also be online in Telegram. But when I open the telegram screen and another window — like this for example:

![Screen Shot 1400-01-25 at 17 38 28](https://user-images.githubusercontent.com/2157285/114715457-538eb680-9d48-11eb-9cc8-a80a0d776dbc.png)

Once I focus on my PDF window, Telegram assumes me to be offline and it shows my status as last seen recently / just now... However I want my friends to assume me online when I have the window opened up like this. Can we have an option in the settings to switch with this default behavior to my idea?

**Comment by d7mtg** [comment]
You're literally not online at that point.

**Comment by pouyakary** [comment]
@d7mtg yes but my window is literally open and I'm feeling online. if someone comes to send me a message I'm literally there but they think I'm not online and they will leave... so you see?
